Dockerfile

```diff
@@ -177,15 +187,11 @@ RUN set -x && \
     # set up auto-gain file structure
     mkdir -p "/run/autogain" && \
     chown readsb "/run/autogain" && \
-    # Install telegraf
-    curl --location --silent -o - https://repos.influxdata.com/influxdb.key | apt-key add - && \
-    source /etc/os-release && \
-    echo "deb https://repos.influxdata.com/debian $VERSION_CODENAME stable" > /etc/apt/sources.list.d/influxdb.list && \
-    apt-get update && \
-    apt-get install --no-install-recommends -y telegraf && \
-    mv -v /etc/telegraf/telegraf.conf /etc/telegraf/telegraf.conf.original && \
+    # Configure telegraf
+    mkdir -p /etc/telegraf && \
```

Comment:
Make this /etc/telegraf/telegraf.d to avoid the problem we had on dump978

Reply:
Oh good catch. Thanks.